astropy/coordinates/solar_system.py

```diff
@@ -274,6 +276,13 @@ def _get_apparent_body_position(body, time, ephemeris):
     cartesian_position : `~astropy.coordinates.CartesianRepresentation`
         Barycentric (ICRS) apparent position of the body in cartesian coordinates
     """
+    # builtin ephemeris and moon is a special case, with no need to account for
```

Comment:
The comment should go two lines below. Also, maybe explicitly note that there is no need to account for light travel time *given* that the effect is smaller than the accuracy of the algorithm? (Or is it included already?)